R/output.R

```diff
@@ -31,7 +31,20 @@ output <- function(rslts, dataformat, fileformat, dirname)
                     msg='output_csv: Invalid results structure, lists nested > 2 deep.')
     }
 
-    if(!fileformat %in% c('CSV', 'XLSX')) {
+    if(fileformat == 'R') {
+        return(rslts)
```

Comment:
This should probably be `NULL` in order to be consistent with the other nodes; however, the return value of `output` is never used, _and_ it is never called in the first place when `fileformat == 'R'`, so we can let it slide.